app/models/BankAccountDetails.scala

```diff
@@ -54,6 +54,8 @@ object BankAccountDetails {
 }
 
 object BankAccount extends JsonUtilities {
+
+  implicit val apiKey: ApiKey[BankAccount] = ApiKey("bankAccount")
```

Comment:
In backend section key for this is configured as `bank-account`, which should match to the api-key here. Also, this should fail the ATs with the keys not matching. So, needs a run on those with these changes. thanks!